docs/src/main/sphinx/admin/opentelemetry.md

```diff
@@ -0,0 +1,133 @@
+to help you analyze application performance and behavior. More information about
+the observability and the concepts involved is available in the [OpenTelemetry
+documentation](https://opentelemetry.io/docs/concepts/).
+
+The integration of OpenTelemetry with Trino enables tracing Trino behavior and
+performance. You can use it to diagnose the overall application as well as
+processing of specific queries or other narrower aspects.
+
+Trino emits trace information from the coordinator and the workers. Trace
+information includes the core system such as the query planner and the
+optimizer, and a wide range of connectors and other plugins.
+
+Trino uses any supplied trace identifiers from client tools across the cluster.
+If none are supplied, trace identifiers are created for each query. The
+identifiers are propagated to data sources, metastores, and other connected
+components. As a result you can use this distributed tracing information to follow
+all the processing flow of a query from a client tool, through the
+coordinator and all workers to the data sources and other integrations.
+
+If you want to receive traces from data sources and other integrations, these
+tools must also support OpenTelemetry tracing and use the supplied identifiers
+from Trino to propagate the context. Tracing must be enabled separately on these
+tools.
+
+## Configuration
+
+Use tracing with OpenTelemetry by enabling it and configuring the endpoint in
+the [config.properties file](config-properties):
+
+```properties
+tracing.enabled=true
+tracing.exporter.endpoint=http://observe.example.com:4317
+```
+
+Tracing is not enabled by default. The exporter endpoint must specify a URL that
+is accessible from the coordinator and all workers of the cluster. The preceding
+example uses a observability platform deployment available by
+HTTP at the host `observe.example.com`, port `4317`.
+
+## Example use
+
+The following steps provide a simple demo setup to run the open source
+observability platform [Jaeger](https://www.jaegertracing.io/) and Trino locally
+in Docker containers.
+
+Create a shared network for both servers called `platform`:
+
+```shell
+docker network create platform
+```
+
+Start Jaeger in the background:
+
+```shell
+docker run -d \
+  --name jaeger \
+  --network=platform \
+  --network-alias=jaeger \
+  -e COLLECTOR_OTLP_ENABLED=true \
+  -p 16686:16686 \
+  -p 4317:4317 \
+  jaegertracing/all-in-one:latest
+```
+
+The preceding command adds Jaeger to the `platform` network with the hostname
+`jaeger`. It also maps the endpoint and Jaeger UI ports.
+
+Create a `config.properties` file that uses the default setup from the Trino
+container, and adds the tracing configuration with the `jaeger` hostname:
+
+```properties
+node-scheduler.include-coordinator=true
+http-server.http.port=8080
+discovery.uri=http://localhost:8080
+tracing.enabled=true
+tracing.exporter.endpoint=http://localhost:4317
```

Comment:
Should this point to the Jaeger host?

Reply:
forgot to copy the local shell script content into the source.. fixed